### DISCUSSION TITLE
Easy way to add New modules

### DISCUSSION THREAD

**Discussion by pepaulo** [JenuelDev/Believers-Sword#85] [Announcements]
I think that is necessary simplify the adiction modules's methode in Believers Sword. Actually the MyBible module is in a .ZIP Format. A SQLite3 compressed in ZIP.  The list of modules can be used as in Personal Registry list. The example: https://www.dropbox.com/s/49zxxi6b5r9ugen/PePauloGiovanni.registry.json

**Comment by JenuelDev** [comment]
Hi! thanks for the suggestion. Can you give more information or details about this one? :)

**Reply by pepaulo** [reply]
I think that the app can use it as a list of module, as example a file style Dot Registry Dot Json.

The data show about modules is from this file.  This is more easy to editing it than one file per module.

**Reply by JenuelDev** [reply]
oh yeah, that's a good idea. I will try to reconstruct it,, give me time and Ill try to set this one.
